integration_test/pg/storage_test.exs

```diff
@@ -33,19 +27,10 @@ defmodule Ecto.Integration.StorageTest do
     assert Postgres.storage_up(correct_params) == {:error, :already_up}
   end
 
-  test "storage up (wrong credentials)" do
-    refute Postgres.storage_up(wrong_user) == :ok
```

Comment:
Let's catch any Postgrex.Error exit and print the error messages in a nice readable way.